eip-xxxx.md

```diff
@@ -0,0 +1,48 @@
+---
+eip: xxxx
+title: Set default gas limit to XX0M
+description: Recommend a new gas limit value for Fusaka and update execution layer client default configs
+author: Sophia Gold (@sophia-gold), Parithosh Jayanthi (@parithoshj), Toni Wahrstätter (@nerolation), Carl Beekhuizen (@CarlBeek), Ansgar Dietrichs (@adietrichs), Dankrad Feist (@dankrad), Alex Stokes (@ralexstokes), Josh Rudolph (@jrudolph), Giulio Rebuffo (@Giulio2002), Storm Slivkoff (@sslivkoff)
+discussions-to: https://ethereum-magicians.org/t/your-discussion-thread
+status: Draft
+type: Core
```

Comment:
```suggestion
type: Informational
```

Since defaults are not part of the protocol, but client configurations, this would need to be informational like EIP-7840.